cfgov/v1/util/util.py

```diff
@@ -143,3 +142,11 @@ def all_valid_destinations_for_request(request):
                             valid_destination_for_request(request, pair[1])]
 
     return valid_destinations
+
+
+def has_active_filters(request, index):
+    for key, param in request.GET.items():
```

Comment:
use a comprehension here.  also, i think the if blocks can be combined using an and.